dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventChangeLogService.java

```diff
@@ -55,16 +56,30 @@ Page<EventChangeLog> getEventChangeLog(
   List<TrackedEntityDataValueChangeLog> getTrackedEntityDataValueChangeLogs(
       TrackedEntityDataValueChangeLogQueryParams params);
 
+  @Deprecated(since = "2.42")
```

Comment:
since we decided [we will remove the endpoint](https://dhis2.slack.com/archives/C05S1NTHBJ9/p1730292188352939?thread_ts=1730290544.389879&cid=C05S1NTHBJ9), it makes sense to remove everything underneath too, unless some other service is using this, which I think it's not the case